docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,424 @@
+```
+
+Here's an example.
+The following Pod has two Containers. Each Container has a request of 0.25 cpu
+and 64MiB (2<sup>26</sup> bytes) of memory Each Container has a limit of 0.5
+cpu and 128MiB of memory. You can say the Pod has a request of 0.5 cpu and 128
+MiB of memory, and a limit of 1 core and 256MiB of memory.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: frontend
+spec:
+  containers:
+  - name: db
+    image: mysql
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+  - name: wp
+    image: wordpress
+    resources:
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+```
+
+## How Pods with resource requests are scheduled
+
+When you create a Pod, the Kubernetes scheduler selects a node for the Pod to
+run on. Each node has a maximum capacity for each of the resource types: the
+amount of CPU and memory it can provide for Pods. The scheduler ensures that,
+for each resource type, the sum of the resource requests of the scheduled
+Containers is less than the capacity of the node. Note that although actual memory
+or CPU resource usage on nodes is very low, the scheduler still refuses to place
+a Pod on a node if the capacity check fails. This protects against a resource
+shortage on a node when resource usage later increases, for example, during a
+daily peak in request rate.
+
+## How Pods with resource limits are run
+
+When the kubelet starts a Container of a Pod, it passes the CPU and memory limits
+to the container runner: Docker or rkt.
+
+When using Docker:
+
+- The `spec.containers[].resources.requests.cpu` is converted to its core value,
+  which is potentially fractional, and multiplied by 1024. This number is used
+  as the value of the
+  [`--cpu-shares`](https://docs.docker.com/engine/reference/run/#/cpu-share-constraint)
+  flag in the `docker run` command.
+
+- The `spec.containers[].resources.limits.cpu` is converted to its millicore value,
+  multiplied by 100000, and then divided by 1000. This number is used as the value
+  of the [`--cpu-quota`](https://docs.docker.com/engine/reference/run/#/cpu-quota-constraint)
+  flag in the `docker run` command.  he [`--cpu-period`] flag is set to 100000,
+   which represents the default 100ms period for measuring quota usage. The
+   kubelet enforces cpu limits if it is started with the
+  [`--cpu-cfs-quota`] flag set to true. As of Kubernetes version 1.2, this flag
+  defaults to true.
+
+- The `spec.containers[].resources.limits.memory` is converted to an integer, and
+  used as the value of the
+  [`--memory`](https://docs.docker.com/engine/reference/run/#/user-memory-constraints)
+  flag in the `docker run` command.
+
+If a Container exceeds its memory limit, it might be terminated. If it is
+restartable, the kubelet will restart it, as with any other type of runtime
+failure.
+
```

Comment:
Add: `If a Container exceeds its memory request, it is likely that its pod might be evicted whenever the node runs out of memory`.